baybe/searchspace/continuous.py

```diff
@@ -29,16 +31,18 @@ class SubspaceContinuous(SerialMixin):
     parameter views.
     """
 
-    parameters: List[NumericalContinuousParameter] = field(
-        validator=lambda _1, _2, x: validate_parameter_names(x)
+    parameters: Tuple[NumericalContinuousParameter, ...] = field(
```

Comment:
Making this exemplary for several places: Sometimes, you use `Tuple[...]` and sometimes `Sequence[...]` for things that are similar to this. What is the reason for not sticking with one of these everywhere?

Reply:
I think it's hard to answer in general without looking at specific examples. Could you show one where you think it's inappropriate?

But this might help: IMO the attributes of a class should be non-generic. If we used `Sequence` as an attribute type, you would not know inside the class body which operations are available, because you don't know if you are effectively working with a list, tuple, etc. On the other hand, each function (including constructors) should be as generic as possible, that is, we don't care if the user passes a list or a tuple, because we convert it internally to the representation we want to work with. Does it make it clearer?

Reply:
Yes, that makes sense :)